client/verta/verta/client.py

```diff
@@ -1559,7 +1596,7 @@ def _get_records(self, response_msg):
         return response_msg.experiment_runs
 
     def _create_element(self, id_):
-        return ExperimentRun(self._conn, self._conf, _expt_run_id=id_)
```

Comment:
This PR has the fortunate consequence of requiring N fewer calls when iterating over `ExperimentRuns`!